src/comments.js

```diff
@@ -129,40 +140,60 @@ function attach(comments, ast, text) {
   comments.forEach(function(comment) {
     decorateComment(ast, comment, text);
 
-    var pn = comment.precedingNode;
-    var en = comment.enclosingNode;
-    var fn = comment.followingNode;
-
-    if (pn && fn) {
-      var tieCount = tiesToBreak.length;
-      if (tieCount > 0) {
-        var lastTie = tiesToBreak[tieCount - 1];
-
-        assert.strictEqual(
-          lastTie.precedingNode === comment.precedingNode,
-          lastTie.followingNode === comment.followingNode
-        );
+    const pn = comment.precedingNode;
```

Comment:
please use full names, otherwise it's really confusing, what the hell is pn, en, fn